Sources/MotorControlUI/TappingStepView.swift

```diff
@@ -0,0 +1,260 @@
+                    .onChanged { touch in
+                        guard !state.initialTap else { return }
+                        Task {
+                            do {
+                                try await state.recorder.start()
+                                withAnimation(.linear(duration: state.motionConfig.duration)) {
+                                    state.progress = 1.0
+                                }
+                            }
+                            catch {
+                                state.result = ErrorResultObject(identifier: state.node.identifier, error: error)
+                            }
+                        }
+                    }
+            )
+    }
+    
+    @ViewBuilder
+    fileprivate func tappingButtons() -> some View {
+        HStack {
+            Spacer()
+            singleTappingButton(target: .left)
+            Spacer()
+            singleTappingButton(target: .right)
+            Spacer()
+        }
+    }
+    
+    @ViewBuilder
+    fileprivate func insideView() -> some View {
+        VStack {
+            StepHeaderView(state)
+            Spacer()
+            countdownDial()
+            Spacer()
+            Text("\(state.tapCount)", bundle: SharedResources.bundle)
+                .foregroundColor(.textForeground)
+                .font(.countdownNumbers)
+            tappingButtons()
+        }
+    }
+    
+    @ViewBuilder
+    func backgroundView() -> some View {
+        ZStack {
+            surveyTint
+            if let image = state.contentNode.imageInfo?.imageName {
+                Image(image, bundle: SharedResources.bundle)
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .opacity(0.4)
+                    .rotation3DEffect(.degrees(state.flippedImage ? 180 : 0), axis: (x: 0, y: 1, z: 0))
+            }
+        }
+        .edgesIgnoringSafeArea(.all)
+    }
+    
+    @ViewBuilder
+    func content() -> some View {
+        GeometryReader { geometry in
+            insideView()
+                .background (
+                    backgroundView()
+                )
+                .coordinateSpace(name: TappingButtonIdentifier.none.rawValue)
+                .onFingerPressedGesture { location, tapDuration in
+                    guard state.initialTap else { return }
+                    state.tappedScreen(uptime: SystemClock.uptime(),
+                                       timestamp: state.recorder.clock.runningDuration(),
+                                       currentButton: .none,
+                                       location: location,
+                                       duration: tapDuration)
+                }
+        }
+    }
+    
+    var body: some View {
+        content()
+            .onAppear {
+                // Reset the countdown animation.
+                resetCountdown()
+                state.audioFileSoundPlayer.vibrateDevice()
```

Comment:
Does the tapping task vibrate on start? Seems like that might be a bit jarring since its suppose to be flat on a table. This feels like a copy/paste error.